bertopic/plotting/_hierarchy.py

```diff
@@ -27,7 +28,11 @@ def visualize_hierarchy(topic_model,
         top_n_topics: Only select the top n most frequent topics
         width: The width of the figure. Only works if orientation is set to 'left'
         height: The height of the figure. Only works if orientation is set to 'bottom'
-
+        optimal_ordering: bool, optional. If True, the linkage matrix will be reordered so that the distance
```

Comment:
Same here. Other than that it looks good. It might take some time for this to be merged as there are currently some issues with the pipeline. I will, however, starting running the checks.